test/unit/xml-exporter/__snapshots__/manuscript-serializer.test.ts.snap

```diff
@@ -0,0 +1,662 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`serializeManuscript serializes manuscript 1`] = `
```

Comment:
The string here should pass our xml validation checks. In future it would be useful to validate this value using code at the unit tests level (fail fast).